<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreadDetailsParticipants</class>
 <widget class="QWidget" name="ThreadDetailsParticipants">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: bold 9pt &quot;heebo&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="AccountProfilePicture">
       <property name="text">
        <string>Profile Picture</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="AccountName">
         <property name="font">
          <font>
           <family>heebo</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="AccountEducationalEmail">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="AccountOrganization">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
